hw/ip/csrng/doc/_index.md

```diff
@@ -38,8 +39,7 @@ This interface allows peripherals to manage CSRNG instances, and request the CSR
 - The IP interfaces with external entropy sources to obtain any required non-deterministic seed material (entropy) and nonces.
     - Requires an external entropy source which is compliant with NIST SP 800-90B, and which also satisfies the requirements for a PTG.2 class physical non-deterministic random number generator as defined in AIS31.
     - Dedicated hardware interface with external entropy source satisfies requirements for `get_entropy_input()` interface as defined in SP 800-90A.
-    - When needed, utilizes the `Block_Cipher_df` derivation function (as defined in SP 800-90A) for assembling seed material.
-      This allows the use of entropy sources which are not full-entropy (less than one bit of entropy per bit generated).
+    - This block does not use a derivation function and requires full entropy from the entropy source.
```

Comment:
The actual NIST doc says the following:
"The use of the derivation function is
optional if either an approved RBG or an entropy source provides full entropy output when entropy input is requested by the DRBG mechanism. Otherwise, the derivation function shall be used."
The added sentence then appears correct to state the actual implementation.

Reply:
Thanks for that reference, this corresponds with my understanding as well.